webpack.config.js

```diff
@@ -84,9 +91,29 @@ const plugins = [
           rv[(new URL(url)).origin] = true;
         });
       }
-      return Object.keys(rv).join(' ');
+      if (connectSrcURLs.length > 0) {
+        connectSrcURLs.forEach((url) => {
+          rv[(new URL(url)).origin] = true;
+        });
+      }
+      if (Object.keys(rv).length > 0) {
+        return Object.keys(rv).join(' ');
+      }
+      return '';
     })(),
     dap_tag: injectDAPTag,
+    script_src: (function () {
+      const rv = {};
+      if (scriptSrcURLs.length > 0) {
+        scriptSrcURLs.forEach((url) => {
+          rv[(new URL(url)).origin] = true;
+        });
+      }
+      if (Object.keys(rv).length > 0) {
+        return Object.keys(rv).join(' ');
+      }
+      return '';
```

Comment:
`return Object.keys(rv).join(' ');` would work fine without the `if` and `return '';` statements

(same comment for line 99)

Reply:
oops yeah